### PR TITLE
bpo-33944: note about the intended use of code in .pth files

### PR DESCRIPTION
**Pull request by native-api**
https://bugs.python.org/issue33944

Automerge-Triggered-By: @matrixise

### CODE REVIEW

File: Doc/library/site.rst
Comment (terryjreedy) [inline Doc/library/site.rst]:
```suggestion
   The primary intended purpose of import lines is to make the
```

**Review by terryjreedy — CHANGES_REQUESTED** [review]
I highly approve of augmenting the doc as an alternative to deprecating this useful feature.  With one optional suggestion, the text looks good according to the discussion.  Another core dev should approve and commit.

File: Doc/library/site.rst
Comment (native-api) [reply Doc/library/site.rst]: I've changed that to "executable lines": "import line" is a bit beside their point. This also makes for consistent terminology: "executable line" is now the official term for the construct.

**Comment by native-api** [comment]
I have made the requested changes; please review again

**Review by terryjreedy — APPROVED** [review]

**Pull request merged**
